seed_identity_store/settings.py

```diff
@@ -150,7 +162,7 @@
 CELERY_CREATE_MISSING_QUEUES = True
 
 CELERY_TASK_SERIALIZER = 'json'
-CELERY_RESULT_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'pickle'
```

Comment:
@KaitCrawford @hodgestar this is fixed and reversed out now given their advice around the function wrapper.